proposals/fleet-desktop-token-rotation.md

```diff
@@ -0,0 +1,64 @@
+# Token Rotation for Fleet Desktop
+
+This file is based on the original proposal described at [#6348](https://github.com/fleetdm/fleet/issues/6348) modified based on a few lessons we learned and the new communication channel between Orbit and the Fleet server introduced in https://github.com/fleetdm/fleet/issues/6851
+
+**Compatibility**
```

Comment:
We need to support/define the following scenario:

New Orbit talking to an old `Server < 4.21` and then have it regularly check if a new `Fleet >= 4.21` has been deployed, so that it exits and starts over (otherwise we need to tell customers to restart all Orbit processes to make token rotation work).

Reply:
+1 indeed, this is noted in the "Orbit > Compatibility" section below (around line 42)

Reply:
Oh missed it. Looks great.